locale/bg-BG.json

```diff
@@ -0,0 +1,149 @@
+{
+  "general": {
+    "language-change": "Променихте езика към {{language}}. Някой промени изискват рестартиране на страницата.",
+    "not-found": "Страницата в която влизате не може да е намерена."
+  },
+  "navbar": {
+    "title": "SCP: SL Сървъри",
+    "server-list": "Списък от сървъри",
+    "all-stats": "Пълна статистика",
+    "all-server-map": "Карта на всички сървъри"
+  },
+  "select": {
+    "placeholder": "Моля изберете едно от посочените",
+    "empty": "Нищо не е избрано"
+  },
+  "filter-option": {
+    "server-search": {
+      "name": "Търсене на сървъри",
+      "placeholder": "Въведете фрази",
+      "search-btn": "Търсете"
+    },
+    "country-filter": "Филтър държави",
+    "hide-empty": "Скрийте празни сървъри",
+    "hide-full": "Скрийте пълни сървъри",
+    "refresh-btn": "Презареди списъка",
+    "refreshing": "Презареждане на списъка...",
+    "yes-no-filter": {
+      "none": "Включи всички",
+      "yes": "Да",
+      "no": "Не",
+      "friendly-fire": "Приятелски Огън",
+      "whitelist": "Частен",
+      "modded": "Моднат",
+      "order-type": {
+        "name": "Тип спрямо хора",
+        "player-desc": "Много хора",
+        "player-asc": "Малко хора",
+        "distance-asc": "Близко по локация",
+        "distance-desc": "Далеч по локация"
+      }
+    }
+  },
+  "server-stats": {
+    "online-users": "Всички играчи: {{count}}",
+    "online-servers": "Онлайн сървъри: {{count}}",
+    "display-users": "Потребители изброени: {{count}}",
+    "display-servers": "Сървъри изброени: {{count}}",
+    "offline-servers": "Офлайн (запазени) сървъри: {{count}}"
+  },
+  "server-list": {
+    "head": {
+      "number": "Номер",
+      "address": "Адрес",
+      "name": "Име",
+      "pastebin": "Pastebin",
+      "players": "Играчи",
+      "distance": "Разстояние"
+    },
+    "loading": "Зареждане на данни от сървър...",
+    "empty": "Няма данни за показване.",
+    "tech": {
+      "yes": "Да",
+      "no": "Не",
+      "country": "Държава",
+      "game-version": "Версия на играта",
+      "friendly-fire": "Приятелски Огън",
+      "whitelist": "Частен"
+    }
+  },
+  "server-info": {
+    "title": "Информация на сървъра",
+    "not-exist": "Сървъра не съществува.",
+    "status": {
+      "name": "Статус",
+      "online": "Онлайн",
+      "offline": "Офлайн"
+    },
+    "address": "Адрес",
+    "players": "Играчи",
+    "pastebin": "Pastebin",
+    "distance": "Разстояние",
+    "statistics": "Статистика на сървър",
+    "daylight-saving-time": "Часът на графиката не се влияе от лятното часово време.",
+    "graph": {
+      "loading": "Зареждане на данни от сървър...",
+      "visitors": "Посетителя",
+      "visitor-suffix": "",
+      "options": {
+        "date-format": "гггг-мм-дд аа чч:мм",

```

Comment:
`date-format` can only come in English. https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Format_Pattern_Examples Refer to this link and modify it.